From ken@novell.com  Fri Mar 16 15:05:04 2007
From: Ken L Johnson <ken@novell.com>
Date: Fri, 16 Mar 2007 10:17:31 -0600
Subject: USB: berry_charge: correct dbg string for second magic command
To: Greg KH <gregkh@suse.de>
Cc: Ken L Johnson <ken@novell.com>
Message-ID: <200703161017.37165.ken@novell.com>

I was testing the berry_charge module with my Blackberry 8700c and had
great success, thanks. Looking at the code for my own learning I noticed
the following cut and paste error... just a nit.

Signed-off-by: Ken L Johnson <ken@novell.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/misc/berry_charge.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/misc/berry_charge.c
+++ b/drivers/usb/misc/berry_charge.c
@@ -69,7 +69,7 @@ static int magic_charge(struct usb_devic
 		return retval;
 	}
 
-	dbg(&udev->dev, "Sending first magic command\n");
+	dbg(&udev->dev, "Sending second magic command\n");
 	retval = usb_control_msg(udev, usb_rcvctrlpipe(udev, 0),
 				 0xa2, 0x40, 0, 1, dummy_buffer, 0, 100);
 	if (retval != 0) {
